lib/scala/tree/src/main/scala/io/github/tetherlessworld/twxplore/lib/tree/stores/Store.scala

```diff
@@ -1,7 +1,9 @@
 package io.github.tetherlessworld.twxplore.lib.tree.stores
 
 import com.google.inject.ImplementedBy
+import edu.rpi.tw.twks.uri.Uri
 import io.github.tetherlessworld.twxplore.lib.geo.models.domain._
+import io.github.tetherlessworld.twxplore.lib.tree.models.domain.{SelectionArea, SelectionGeometry, SelectionInput, SelectionResults}
 
 @ImplementedBy(classOf[TwksStore])
 trait Store {
```

Comment:
This needs to be broken up into multiple stores (CityStore, BoroughStore, et al.). I'll do that, though.